docs/site/sidebars/lb4_sidebar.yml

```diff
@@ -223,8 +223,28 @@ children:
     children:
 
     - title: 'Authentication'
-      url: Loopback-component-authentication.html
+      url: Authentication-overview.html
       output: 'web, pdf'
+      children:
+        - title: 'Apply JWT Authentication in Todo Example'
+          url: Authentication-tutorial.html
+          output: 'web, pdf'
+
+        - title: 'Authentication Component'
```

Comment:
note: use https://github.com/strongloop/loopback-next/blob/master/docs/site/Loopback-component-authentication.md

Reply:
- [ ] DRAFT PR https://github.com/strongloop/loopback-next/pull/5641, will break it down into smaller ones.